lib/fog/openstack/models/volume/volume.rb

```diff
@@ -42,6 +44,13 @@ def extend(size)
         def ready?
           status == 'available'
         end
+
+        def display_name
+          name
+        end
+        def display_description
+          description
+        end
```

Comment:
This is also obviously ugly. The `spec/fog/openstack/volume_spec.rb` uses these accessors, but giving `display_name` and `display_description` as aliases for `name` and `description` above does not seem to generate them. Again, is there a nicer way to handle this?